src/common/popup/utils/format.js

```diff
@@ -14,8 +14,13 @@ function branch(ticket) {
   return `${ticket.type || 'feature'}/${ticket.id}-${normalize(ticket.title)}`;
 }
 
+function command(ticket) {
+  return `git checkout -b ${branch(ticket)}  && git commit --allow-empty -m"${commit(ticket)}"`;
```

Comment:
There should be a space between `-m` and the message. It'll work like this, but for extra style points… 😝 (https://git-scm.com/docs/git-commit) (Sorry for nitpicking).